src/faqs/items.njk

```diff
@@ -15,16 +15,17 @@ eleventyComputed:
 <section class="px-4 lg:container mx-auto">
   <a href="/{{ locale }}/faqs" class="flex items-center mb-2 text-red hover:text-red-dark md:mb-4">
     {% svgSprite name="arrow-left", class="h-6 w-6 hover:fill-current" %}
-    <span class="pl-1">{{ lang.backButton}}</span>
+    <span class="pl-1">{{ lang.backButton }}</span>
   </a>
 
-
   <div class="max-w-3xl">
     <h1 class="title mb-2">{{ title | safe }}</h1>
     <div class="alt-prose">
-      {{ item.detailedAnswer|markdown|safe }}
+      {% if item.detailedAnswer %}
+        {{ item.detailedAnswer | markdown | safe }}
+      {% else %}
```

Comment:
Since we are only using the `detailedAnswer` on FAQs details page, why not leave the `description` piece out? The user will not access the page and it will not be generated by Eleventy.